### PR TITLE
Add boss Legion.json

### PR DESCRIPTION
**Pull request by devilflippy**
We could nerf his experience reward for kill.
Fucking high right now

### CODE REVIEW

File: user/mods/RaidOverhaul/db/RaidBoss/bosslegion2.json
Comment (guillaumearm) [inline user/mods/RaidOverhaul/db/RaidBoss/bosslegion2.json:1]: Why this one ? is it used ?

Comment (devilflippy) [reply user/mods/RaidOverhaul/db/RaidBoss/bosslegion2.json:1]: in Raid Overhaul yeah

Comment (guillaumearm) [reply user/mods/RaidOverhaul/db/RaidBoss/bosslegion2.json:1]: I mean there is `bosslegion.json` and `bosslegion2.json` what's the difference here ?

Comment (devilflippy) [reply user/mods/RaidOverhaul/db/RaidBoss/bosslegion2.json:1]: idk

**Review by guillaumearm — APPROVED** [review]

**Pull request merged**